packages/ubuntu_bootstrap/lib/installer.dart

```diff
@@ -108,6 +109,9 @@ Future<void> runInstallerApp(
     () => PageConfigService(
       config: tryGetService<ConfigService>(),
       includeTryOrInstall: includeTryOrInstall,
+      allowedToHide: InstallationStep.values
+          .where((e) => e.allowedToHide)
+          .map((e) => e.route),
```

Comment:
```suggestion
          .map((e) => e.name),
```

`_excludedPages` in `PageConfigService` uses the names of the pages without the leading '/'

Reply:
Wtf... How did this even work...? :sweat_smile: 
Good catch!